### ISSUE TITLE
Generating new training data

### ISSUE THREAD

**Issue by chaine09** [jaeoh2/Road-Lane-Instance-Segmentation-PyTorch#1]
Have you tried generating your own dataset similar to Tusimple Dataset?

If yes, how?

What's the proper format for both binary and instance segmentation image labels?

**Comment by jaeoh2** [comment]
No I didn't tried in custom dataset.
If you want to test your own dataset, you need line-by-line instance segmentation label from imageset.

You can try labeling tools below.
https://github.com/ucbdrive/scalabel
https://github.com/opencv/cvat
